src/utils/init-project/sushi-config.yaml

```diff
@@ -1,6 +1,6 @@
 # ╭──────────────────────────────────────ImplementationGuide───────────────────────────────────────╮
 # │  The properties below are used to create the ImplementationGuide resource. For a list of       │
-# │  supported properties, see: https://fshschool.org/sushi/configuration/                         │
+# │  supported properties, see: https://fshschool.org/docs/sushi/configuration/                         │
```

Comment:
I don't believe this actually affects the tests in any meaningful way, but the new link is a few characters longer than the old, so the `|` that closes the box on this line is a little too far out to correctly form the box. Would you mind removing a few of the trailing spaces between the link and the `|` so the boxes lines up here? Also, if you wouldn't mind, would you be able to make that change in the other two test fixtures below this one?

Reply:
@masnick -- is this something you can quickly fix for us, or would you prefer we just pop in and do it?  I'm not sure if we have permission to push to your fork or not...